### DISCUSSION TITLE
Phoenix Questionnaire Builder vs. View

### DISCUSSION THREAD

**Discussion by zoyagarg** [CS342/.github#56] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

Hi! Team Intake just figured out how to extract health data and build LLM chat functionality -- one thing we're a little confused about from here is if we need to use Phoenix Questionnaire Builder at all or if we can just use the health data formatted in view?

### Reproduction

No bugs so far, just would love advice about technical strategy?

### Expected behavior

N/A

### Additional context

Thank you!

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by philippzagar** [comment, marked as answer]
Hi @zoyagarg, thanks for creating the GitHub issue! 🚀 
The [Phoenix Questionnaire Builder](https://github.com/StanfordBDHG/phoenix) as well as [SpeziQuestionnaire](https://github.com/StanfordSpezi/SpeziQuestionnaire) only support questionnaire types supported by [Apple's ResearchKit](https://researchkit.org/). As the questionnaire types provided by ResearchKit are quite limited, I would suggest to either build something on your own in SwiftUI (for the respective form type) or use Spezi modules for a specific use case like [SpeziMedication](https://github.com/StanfordSpezi/SpeziMedication).

**Reply by PSchmiedmayer** [reply]
@zoyagarg I assume this comment by @philippzagar resolved your question? If not, do you have any follow-up questions?

**Reply by zoyagarg** [reply]
yep, thank you so much!
